src/transaction.spec.ts

```diff
@@ -696,4 +697,60 @@ describe("test transaction", async () => {
         assert.equal(transaction.version, 2);
         assert.equal(transaction.options, 3);
     });
+
+    it("should compute bytes to verify transaction signature", async () => {
+        let transaction = new Transaction({
+            sender: wallets.alice.address.toBech32(),
```

Comment:
For the sake of the examples, we can also add a transaction nonce - here and below.

Reply:
added